### PR TITLE
Fb1175

### PR DESCRIPTION
**Pull request by spireken**
I put the build in the book_view_window folder on Dropbox.

### CODE REVIEW

File: Applications/spire/source/book_view/bbo_quote_panel.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using namespace Spire;
+
+BboQuotePanel::BboQuotePanel(const BookViewModel& model,
+    QWidget* parent)
+    : QWidget(parent) {
+  setFixedHeight(scale_height(28));
+  auto layout = new QHBoxLayout(this);
+  layout->setContentsMargins({});
+  layout->setSpacing(0);
+  m_bid_quote_panel = new QuotePanel(model, Nexus::Side::BID, this);
```
Comment (kranar) [inline Applications/spire/source/book_view/bbo_quote_panel.cpp]: Once you pull in namespace Nexus, you can remove Nexus::

File: Applications/spire/source/book_view/quote_panel.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+void QuotePanel::on_bbo_quote(const BboQuote& bbo) {
+  auto& quote = [&] {
+    if(m_side == Side::BID) {
+      return bbo.m_bid;
+    }
+    return bbo.m_ask;
+  }();
+  if(quote.m_price > m_current_bbo.m_bid.m_price) {
```
Comment (kranar) [inline Applications/spire/source/book_view/quote_panel.cpp]: You need to compare bid to bid, ask to ask. Here's you're always comparing to bid.

**Comment by spireken** [comment]
I updated the build on Dropbox.

**Comment by spireken** [comment]
Actually, I think I can do this better.

**Pull request closed without merging**